datasae/string.py

```diff
@@ -0,0 +1,575 @@
+#!/usr/bin/env python3
+
+# This module is part of DataSae and is released under
+# the AGPL-3.0-only License: https://opensource.org/license/agpl-v3/
+
+
+# !/usr/bin/env python3
+
+# This module is part of DataSae and is released under
+# the AGPL-3.0-only License: https://opensource.org/license/agpl-v3/
+
+
+from .exception import InvalidDataTypeWarning, InvalidDataValueWarning
+from .utils import Basic, create_warning_data, WarningDataMessage
+import pandas as pd
+import re
```

Comment:
Ubah jadi gini paragrah import nya @riskaamalia 

ini untuk dikelompokin paragraph import jadi:
1. built-in package
2. install package
3. our package
```py
import re

import pandas as pd

from .exception import InvalidDataTypeWarning, InvalidDataValueWarning
from .utils import Basic, create_warning_data, WarningDataMessage
```